hedera-node/hedera-token-service-impl/src/main/java/module-info.java

```diff
@@ -17,7 +17,8 @@
 
     exports com.hedera.node.app.service.token.impl to
             com.hedera.node.app.service.token.impl.test,
-            com.hedera.node.app;
+            com.hedera.node.app,
+            com.hedera.node.app.service.contract.impl;
```

Comment:
This should never happen. By doing so the implementation of a service depends on the implementation of another service

Reply:
Yes that got added when I was trying out if I can add something in contracts module. Removed it. Thank you !

Reply:
@hendrikebbers sounds good.